pom.xml

```diff
@@ -153,6 +153,110 @@
+                <id>Copy Central Auth and Monitoring Resources </id>
+                <phase>process-resources</phase>
+                <goals>
+                  <goal>copy-resources</goal>
+                </goals>
+                <configuration>
+                  <outputDirectory>
+                    ${project.build.directory}/${project.artifactId}-${project.version}/distro/binaries/
+                  </outputDirectory>
+                  <overwrite>true</overwrite>
+                  <resources>
+                    <resource>
+                      <targetPath>
+                        ${project.build.directory}/${project.artifactId}-${project.version}/distro/binaries/eip-keycloak-superset</targetPath>
+                      <directory>
+                        ${project.build.directory}/ozone-pro/distro/binaries/eip-keycloak-superset/</directory>
+                    </resource>
+                    <resource>
+                      <targetPath>
+                        ${project.build.directory}/${project.artifactId}-${project.version}/distro/binaries/eip-keycloak-openmrs</targetPath>
+                      <directory>
+                        ${project.build.directory}/ozone-pro/distro/binaries/eip-keycloak-openmrs/</directory>
+                    </resource>
+                    <resource>
+                      <targetPath>
+                        ${project.build.directory}/${project.artifactId}-${project.version}/distro/binaries/openmrs/jmx_exporter</targetPath>
+                      <directory>
+                        ${project.build.directory}/ozone-pro/distro/binaries/openmrs/jmx_exporter</directory>
+                    </resource>
+                  </resources>
+                </configuration>
+              </execution>
+              <execution>
+                <!-- Copy Central Auth and Monitoring Docker Resources -->
+                <id>Copy Central Auth and Monitoring Docker Resources</id>
+                <phase>process-resources</phase>
+                <goals>
+                  <goal>copy-resources</goal>
+                </goals>
+                <configuration>
+                  <outputDirectory>
+                    ${project.build.directory}/${project.artifactId}-${project.version}/run/docker/
+                  </outputDirectory>
+                  <overwrite>true</overwrite>
+                  <resources>
+                    <resource>
+                      <directory>
+                        ${project.build.directory}/ozone-pro/run/docker</directory>
+                      <includes>
+                        <include>docker-compose-common-pro.yml</include>
+                        <include>docker-compose-eip-keycloak-openmrs.yml</include>
+                        <include>docker-compose-eip-keycloak-superset.yml</include>
+                        <include>docker-compose-openmrs-pro.yml</include>
+                      </includes>
```

Comment:
`docker-compose-eip-keycloak-odoo.yml` is missing? Does Ozone HSC uses Odoo ERP?

Reply:
I will add that. I was not aware of it